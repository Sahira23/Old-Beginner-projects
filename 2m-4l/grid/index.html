<!DOCTYPE html>
<html lang="en">
<head>
    <!-- !REQUIRED META TAGS -->
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Grid</title>
    <!-- !STYLE CSS -->
    <link rel="stylesheet" href="./css/style.css">
</head>
<body>
    <div class="grid-container">
      <div class="grid-item-1 grid-item">HEADER</div>
      <div class="grid-item-2 grid-item">MENU</div>
      <div class="grid-item-3 grid-item">ASIDE</div>
      <div class="grid-item-4 grid-item">FOOTER</div>  
    </div>
</body>
</html>