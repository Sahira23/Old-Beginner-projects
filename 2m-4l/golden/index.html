<!DOCTYPE html>
<html lang="en">
  <head>
    <!-- !REQUIRED META TAGS -->
    <meta charset="UTF-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>GOLDEN</title>
    <!-- !STYLE CSS -->
    <link rel="stylesheet" href="./css/style.css" />
  </head>
  <body>
    <!-- !container -->
    <div class="container">
      <!-- !HEADER -->
      <header class="useless">
        <!-- !NAV-BAR -->
        <nav class="d-flex justify-between">
          <!-- !LOGO -->
          <div class="logo d-flex align-center">
            <img src="./images/logo.png" alt="logo" />
          </div>
          <!-- !LINKS -->
          <div class="links">
            <a href="#">HOME</a>
            <a href="#">SERVICES</a>
            <a href="#">PORTFOLIO</a>
            <a href="#">ABOUT</a>
            <a href="#">CONTACT</a>
          </div>
        </nav>
        <!-- !HOME -->
        <div class="home d-flex direction-column justify-center align-center">
          <h2>Welcome To Our Studio!</h2>
          <h1>Itâ€™s nice to Meet you</h1>
          <button type="button">Tell Me More</button>
        </div>
      </header>
      <section>
        <!-- !ARTICLE--SERVICES -->
        <article class="services useless">
          <!-- !HEAD -->
          <div class="head d-flex direction-column align-center">
            <h2>SERVICES</h2>
            <p>Proin iaculis purus consequat sem cure.</p>
          </div>
          <!-- !COMPONENTS -->
          <div class="components d-flex justify-between">
            <!-- !E-COMMERCE -->
            <div class="e-commerce d-flex direction-column align-center">
              <div class="image d-flex align-center justify-center">
                <img src="./images/e-commerce.png" alt="e-commerce" />
              </div>
              <h4>E-Commerce</h4>
              <p>
                Proin iaculis purus consequat sem cure digni ssim. Donec
                porttitora entum suscipit aenean rhoncus posuere odio in
                tincidunt.
              </p>
            </div>
            <!-- !RESPONSIVE DESIGN -->
            <div class="responsive-design d-flex direction-column align-center">
              <div class="image d-flex align-center justify-center">
                <img src="./images/responsive-design.png" alt="e-commerce" />
              </div>
              <h4>Responsive Web</h4>
              <p>
                Proin iaculis purus consequat sem cure digni ssim. Donec
                porttitora entum suscipit aenean rhoncus posuere odio in
                tincidunt.
              </p>
            </div>
            <!-- !WEB SECURITY -->
            <div class="web-security d-flex direction-column align-center">
              <div class="image d-flex align-center justify-center">
                <img src="./images/security.png" alt="e-commerce" />
              </div>
              <h4>Web Security</h4>
              <p>
                Proin iaculis purus consequat sem cure digni ssim. Donec
                porttitora entum suscipit aenean rhoncus posuere odio in
                tincidunt.
              </p>
            </div>
          </div>
        </article>
        <!-- !ARTICLE--OUR PORTFOLIO -->
        <article class="our-portfolio useless">
          <!-- !HEAD -->
          <div class="head d-flex direction-column align-center">
            <h2>OUR PORTFOLIO</h2>
            <p>Proin iaculis purus consequat sem cure.</p>
          </div>
          <!-- !ROW1 CARDS -->
          <div class="row1 d-flex justify-between">
            <!-- !1 -->
            <div class="card">
              <img src="./images/elbony.jpg" alt="elbony-ivory" />
              <h5>Ebony & Ivory</h5>
              <p>Branding</p>
            </div>
            <!-- !2 -->
            <div class="card">
              <img src="./images/smart.jpg" alt="smart stationary" />
              <h5>Smart Stationary</h5>
              <p>Print Design</p>
            </div>
            <!-- !3 -->
            <div class="card">
              <img src="./images/clever.jpg" alt="clever poster" />
              <h5>Clever Poster</h5>
              <p>Print Design</p>
            </div>
          </div>
          <!-- !ROW2 CARDS  -->
          <div class="row2 d-flex justify-between">
            <!-- !1 -->
            <div class="card">
              <img src="./images/vinylrecord.jpg" alt="" />
              <h5>Vinyl Record</h5>
              <p>Product Mock-Up</p>
            </div>
            <!-- !2 -->
            <div class="card">
              <img src="./images/treehouse.jpg" alt="" />
              <h5>Treehouse Template</h5>
              <p>Web Design</p>
            </div>
            <!-- !3 -->
            <div class="card">
              <img src="./images/burnedlogo.jpg" alt="" />
              <h5>Burned Logo</h5>
              <p>Branding</p>
            </div>
          </div>
        </article>
        <!-- !ARTICLE--ABOUT US -->
        <article class="about-us useless">
          <!-- !HEAD -->
          <div class="head d-flex direction-column align-center">
            <h2>ABOUT US</h2>
            <p>Proin iaculis purus consequat sem cure.</p>
          </div>
          <!-- !MONTHLY-GOALS -->
          <div class="month">
            <!-- !1 -->
            <div class="row-1 d-flex">
              <!-- !TEXT -->
              <div class="text text-right mr-md">
                <h3>July 2010</h3>
                <h3>Our Humble Beginnings</h3>
                <p>
                  Proin iaculis purus consequat sem cure digni ssim. Donec
                  porttitora entum suscipit aenean rhoncus posuere odio in
                  tincidunt. Proin iaculis purus consequat sem cure digni ssim.
                  Donec porttitora entum suscipit.
                </p>
              </div>
              <!-- !IMAGE -->
              <div class="about-us-image d-flex direction-column align-center">
                <img src="./images/july.jpg" alt="july" />
                <img src="./images/line.png" alt="line" />
              </div>
            </div>
            <!-- !2 -->
            <div class="row-2 d-flex justify-end">
              <!-- !IMAGE -->
              <div
                class="
                  about-us-image
                  d-flex
                  direction-column
                  align-center
                  mr-md
                "
              >
                <img src="./images/january.jpg" alt="january" />
                <img src="./images/line.png" alt="line" />
              </div>
              <!-- !TEXT -->
              <div class="text text-left">
                <h3>January 2011</h3>
                <h3>Facing Startup Battles</h3>
                <p>
                  Proin iaculis purus consequat sem cure digni ssim. Donec
                  porttitora entum suscipit aenean rhoncus posuere odio in
                  tincidunt. Proin iaculis purus consequat sem cure digni ssim.
                  Donec porttitora entum suscipit.
                </p>
              </div>
            </div>
            <!-- !3 -->
            <div class="row-3 d-flex">
              <!-- !TEXT -->
              <div class="text text-right mr-md">
                <h3>December 2012</h3>
                <h3>Enter The Dark Days</h3>
                <p>
                  Proin iaculis purus consequat sem cure digni ssim. Donec
                  porttitora entum suscipit aenean rhoncus posuere odio in
                  tincidunt. Proin iaculis purus consequat sem cure digni ssim.
                  Donec porttitora entum suscipit.
                </p>
              </div>
              <!-- !IMAGE -->
              <div class="about-us-image d-flex direction-column align-center">
                <img src="./images/december.jpg" alt="december" />
                <img src="./images/line.png" alt="line" />
              </div>
            </div>
            <!-- !4 -->
            <div class="row-4 d-flex justify-end">
              <!-- !IMAGE -->
              <div
                class="
                  about-us-image
                  d-flex
                  direction-column
                  align-center
                  mr-md
                "
              >
                <img src="./images/february.jpg" alt="february" />
                <img src="./images/line.png" alt="line" />
              </div>
              <!-- !TEXT -->
              <div class="text text-left">
                <h3 february 2014></h3>
                <h3>Our Triumph</h3>
                <p>
                  Proin iaculis purus consequat sem cure digni ssim. Donec
                  porttitora entum suscipit aenean rhoncus posuere odio in
                  tincidunt. Proin iaculis purus consequat sem cure digni ssim.
                  Donec porttitora entum suscipit.
                </p>
              </div>
            </div>
            <!-- !5 -->
            <div class="row-5 d-flex justify-center">
              <div class="continue d-flex align-center">
                <h4>Our Story Continues ...</h4>
              </div>
            </div>
          </div>
        </article>
        <!-- !ARTICLE --OUR TEAM -->
        <article class="our-team useless">
          <!-- !HEAD -->
          <div class="head d-flex direction-column align-center">
            <h2>OUR AMAZING TEAM</h2>
            <p>Proin iaculis purus consequat sem cure.</p>
          </div>
          <!-- !FIRST ROW -->
          <div class="first-row d-flex justify-around">
            <!-- !MARKETER -->
            <div class="marketer">
              <div class="profile"></div>
              <div class="name">
                <h4>Kimberly Thompson</h4>
                <p>Marketer</p>
              </div>
              <!-- !SOCIAL-MEDIA-ICONS -->
              <div class="social-icons d-flex justify-center">
                <div class="icon d-flex align-center justify-center">
                  <img src="./images/twitter.png" alt="twitter" />
                </div>
                <div class="icon d-flex align-center justify-center">
                  <img src="./images/facebook.png" alt="facebook" />
                </div>
                <div class="icon d-flex align-center justify-center">
                  <img src="./images/pinterest.png" alt="pinterest" />
                </div>
              </div>
            </div>
            <!-- !CODER -->
            <div class="coder">
              <div class="profile"></div>
              <div class="name">
                <h4>Rico Massimo</h4>
                <p>Coder</p>
              </div>
                <!-- !SOCIAL-MEDIA-ICONS -->
              <div class="social-icons d-flex justify-center">
                <div class="icon d-flex align-center justify-center">
                  <img src="./images/twitter.png" alt="twitter" />
                </div>
                <div class="icon d-flex align-center justify-center">
                  <img src="./images/facebook.png" alt="facebook" />
                </div>
                <div class="icon d-flex align-center justify-center">
                  <img src="./images/googleplus.png" alt="googlepus" />
                </div>
              </div>
            </div>
            <!-- !DESIGNER -->
            <div class="designer">
              <div class="profile"></div>
              <div class="name">
                <h4>Uku Mason</h4>
                <p>Graphic Designer</p>
              </div>
                <!-- !SOCIAL-MEDIA-ICONS -->
              <div class="social-icons d-flex justify-center">
                <div class="icon d-flex align-center justify-center">
                  <img src="./images/facebook.png" alt="facebook" />
                </div>
                <div class="icon d-flex align-center justify-center">
                  <img src="./images/pinterest.png" alt="pinterest" />
                </div>
              </div>
            </div>
          </div>
          <p>
            Proin iaculis purus consequat sem cure digni ssim donec porttitora
            entum suscipit aenean rhoncus posuere odio in tincidunt proin
            iaculis.
          </p>
        </article>
        <!-- !ARTICLE--LOGOS -->
        <article class="logos useless d-flex justify-around">
          <img src="./images/envato.png" alt="envato" />
          <img src="./images/wordpress.png" alt="wordpress" />
          <img src="./images/tuts.png" alt="tuts" />
          <img src="./images/microlancer.png" alt="microlancer" />
        </article>
        <!-- !CONTACT-US -->
        <article class="contact useless d-flex direction-column align-center">
          <!-- !HEAD -->
          <div class="head d-flex direction-column align-center">
            <h2>CONTACT US</h2>
            <p>Proin iaculis purus consequat sem cure.</p>
          </div>
          <!-- !FORM -->
          <div class="form d-flex ">
            <!-- !FIRST-COLUMN -->
          <div class="input-side d-flex direction-column">
           <input type="text" name="name" placeholder="your Name *" >
           <input type="email" name="email" placeholder="your e-mail *">
           <input type="text" name="subject" placeholder="subject">
          </div>
          <!-- !SECOND-COLUMN -->
          <div class="message-side">
            <input type="text" name="message" placeholder="Your message *" >
          </div>
        </div>
        <button type="button">
          SEND MESSAGE
          </button>
        </article>
      </section>
      <!-- !FOOTER -->
      <footer class="useless d-flex align-center">
       <p>Â© Copyright 2014 FreebiesXpress.com</p>
       <div class="social-links">
         <img src="./images/foot-twitter.png" alt="twitter">
         <img src="./images/foot-facebook.png" alt="facebook">
         <img src="./images/foot-pinterest.png" alt="pinterest">
         <img src="./images/foot-googleplus.png" alt="googleplus">
       </div>
      </footer>
    </div>
  </body>
</html>
